nncf/onnx/quantization/default_quantization.py

```diff
@@ -9,10 +9,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from nncf.common.graph.operator_metatypes import UnknownMetatype
 from nncf.common.quantization.quantizer_propagation.structs import QuantizationTrait
 from nncf.onnx.graph.metatypes import onnx_metatypes
 
+# If there are no some metatypes it means that they are considered as QuantizationTrait.NON_QUANTIZABLE
```

Comment:
`If a metatype is not in this list, then it is considered to be QuantizationTrait.NON_QUANTIZABLE.`